src/components/common-components/route-buttons/route-buttons.component.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { Tab, withStyles } from "@material-ui/core";
+import TabContext from "@material-ui/lab/TabContext";
+import TabList from "@material-ui/lab/TabList";
+import TabPanel from "@material-ui/lab/TabPanel";
+import { makeStyles } from "@material-ui/core/styles";
+import ScssVars from "../../../assets/styles/styles/custom/route-buttons.module.scss";
+
+interface RouteButtonsComponent {
+  components: { [key: string]: React.FC };
+}
+
+const useStyles = makeStyles(() => ({
+  indicatorStyle: {
+    backgroundColor: ScssVars.ic,
+    height: 3,
+    opacity: 1,
+    outline: "none",
+  },
+
+  tabStyle: {
+    opacity: 1,
+    outline: "none",
+  },
+}));
+
+const StyledTab: any = withStyles((theme) => ({
+  root: {
+    textTransform: "none",
+    color: ScssVars.tc,
+    outline: "none",
+    fontWeight: theme.typography.fontWeightMedium,
+    fontSize: "16",
+    opacity: 1,
+    fontFamily: ["Roboto"].join(","),
+    "&:hover": {
+      color: ScssVars.ic,
+      opacity: 1,
+      outline: "none",
+    },
+    "&$selected": {
+      color: ScssVars.tc,
+      outline: "none",
+      fontWeight: theme.typography.fontWeightBold,
+    },
+  },
+
+  selected: {
+    outline: "none",
+  },
+}))((props) => <Tab disableRipple {...props} />);
```

Comment:
@pectom, myśę, że na razie możemy żyć z takim stylowaniem, ponieważ nie jest to rzecz krytyczna. a może zająć dużo czasu .

Reply:
Jakby coś, to dodałem też task na tarze na zmianę tego https://app.tara.ai/nurse-scheduling/tasks/120

Reply:
Ehh jakoś przeżyjemy :D Popieram, rozdział tego do innego taska @VVlovsky odwalił kawał dobrej roboty i fajnie by to było już mieć na developie. Czekamy, więc na te małe poprawki i mergujemy :)

Reply:
@pectom ogólnie też niestety nie jestem zadowolony takim stylowaniem, już omawialiśmy to z Bohdanem, wydaje mi się że wykorzystywanie komponentów Material UI jest zbędne w przypadku gdy komponent od material się różni od tego który chcemy zaimplementować więcej niż kolorom, ponieważ materiał choć i posiada instrumenty do stylowania, ale nie przewiduje robienie tego przez pliki scss i nie przewiduje tą strukturę projektu, którą my realizujemy. Chciałbym zaimplementować taby które będą niezależne od material ui i będą wykorzystywać pliki scss, lecz takie zadanie jest teraz bardzo słabe w porównaniu z innymi pod względem stosunku otrzymanego rezultatu do czasu. Co o tym myślisz?

Co do PR, to w ciągu kilku godzin wprowadzę wszystkie potrzebne zmiany, dziękuję za review :)

Reply:
Z Agą - naszą grafik - ustaliliśmy, że będziemy korzystać z komponentów z Materiala, więc na pewno potrzebujemy mieć możliwość ich stylowania w wygodny sposób. Bardziej byłbym skłonny do zrezygnowania z plików scss niż z Materiala. 

Moim zdaniem nie ma co zajmować się samymi tabami, jeśli jesteś zainteresowany tą sprawą to mógłbyś zrobić research jak wygodnie stylować materiala i czy w jakiś sposób jednak da się go połączyć z scss. Jeśli będzie to niemożlwie to musimy ustalić wspólnie jakąś konwencję. Mógłbyś w takim wypadku coś zaproponować i wtedy razem z @Qwebeck to dogramy.